### PR TITLE
workspacekit: Record the errors of connecting to ws-daemon

### PR DESCRIPTION
**Pull request by utam0k**
## Description
workspacekit: Record the errors of connecting to ws-daemon

## Related Issue(s)

Relates: #9673

## How to test

Open a workspace

## Release Notes

```release-note
None
```

## Documentation

No

### CODE REVIEW

File: components/workspacekit/cmd/rings.go [hunk 1]
```diff
@@ -917,17 +918,20 @@ func connectToInWorkspaceDaemonService(ctx context.Context) (*inWorkspaceService
 	const socketFN = "/.workspace/daemon.sock"
 
 	t := time.NewTicker(500 * time.Millisecond)
+	errs := errors.New("errors of connect to ws-daemon")
 	defer t.Stop()
 	for {
 		if _, err := os.Stat(socketFN); err == nil {
 			break
+		} else if !os.IsNotExist(err) {
+			errs = fmt.Errorf("%v; %w", errs, err)
```
Comment (jenting) [inline components/workspacekit/cmd/rings.go:927]: nit
```suggestion
                        errs = fmt.Errorf("%v: %w", errs, err)
```

Comment (utam0k) [reply components/workspacekit/cmd/rings.go:927]: @jenting Thanks for your review. I'm a beginner of go language but please let me why `:` is better than `;`.

Comment (jenting) [reply components/workspacekit/cmd/rings.go:927]: It's not language specific, it's the coding convention.
Mostly, the error message would be wrap as `xxx: yyy`.

**Comment by utam0k** [comment]
@csweichel @jenting PTAL

**Review by jenting — APPROVED** [review]
/lgtm

**Pull request merged**